application/org.openjdk.jmc.flightrecorder.hierarchical-edge/src/main/resources/hierarchical-edge.js

```diff
@@ -0,0 +1,351 @@
+function updateGraph(eventsJson) {
```

Comment:
First part here is indented with tabs. The rest with spaces. JMC uses 1-tab, 1-level-indent. (Yup, I know that we may be failing this in other parts of our JS code.)